src/ui/view/vault.rs

```diff
@@ -223,9 +223,57 @@ impl VaultOnChainTransactionsPanel {
                             .align_items(Align::Center),
                     )))
                 }
+                VaultStatus::Unvaulted | VaultStatus::Unvaulting => {
+                    col = col.push(card::white(Container::new(
+                        Row::new()
+                            .push(
+                                Container::new(text::simple(
+                                    "Funds are moving, do you want to revault it ?",
```

Comment:
nit: maybe "do you want to revault *them?*" (also please remove the space before the ?)